src/operator/contrib/adaptive_avg_pooling.cc

```diff
@@ -197,6 +198,91 @@ num_threads(engine::OpenMP::Get()->GetRecommendedOMPThreadCount())
   }
 }
 
+#if MXNET_USE_MKLDNN == 1
+bool CanMKLDNNSupportAveragePooling(const NDArray &in_data,
```

Comment:
Please change the name to SupportMKLDNNAveragePooling or SupportMKLDNNAP to stay consistent with other implementations.

Reply:
Done.